code/game/jobs/job/civilian_chaplain.dm

```diff
@@ -410,6 +410,11 @@
 				H.equip_or_collect(new /obj/item/clothing/mask/gas/mime(H), slot_wear_mask)
 				J = "..."
 				deity_name = "Silence"
+			if("Ancap", "Ancapistan", "NAP")
```

Comment:
No really, add 
qdel(B)
B = new /obj/item/weapon/storage/bible/booze(H)
right under this line
Gives the chaplain the booze and money bible